### DISCUSSION TITLE
Reconstruct 3D Mesh and export ?

### DISCUSSION THREAD

**Discussion by darkworks** [CERN/TIGRE#540] [Q&A]
Am wondering is it possible to reconstruct 3D mesh from 2D xray tiff images take via x ray scanner like these
![image](https://github.com/CERN/TIGRE/assets/12841788/da3e4f72-e32e-4d09-ac09-5298dca23f77)

and then export it to stl or dxf or other format to visualize geometry in 3d software 

currently am trying with tomviz but algos are disabled in it not sure why ? : https://github.com/OpenChemistry/tomviz/issues/2295
also try tomopy but it not support 3d export : https://github.com/tomopy/tomopy/issues/642
so am here to try out TIGRE.

thanks

**Comment by AnderBiguri** [comment]
I can't speak about tomviz or tomopy (albeit the last should work). Yes, it is possible, given a known geometry for each projection, to recosntruct, and the save the volume whichever way you want. Of course no CT recon produces a STL, that would require a secondary step of segmentation and surface extraction, which you would need to code yourself or use another software for it.
